jdaviz/configs/default/plugins/export/tests/test_export.py

```diff
@@ -218,6 +219,45 @@ def test_basic_export_subsets_cubeviz(self, cubeviz_helper, spectral_cube_wcs):
         export_plugin.export()
         assert os.path.isfile('test_spectral_region.ecsv')
 
+    def test_export_stcs_circle_ellipse(self, imviz_helper):
+        wcs = WCS({'CTYPE1': 'RA---TAN', 'CUNIT1': 'deg', 'CDELT1': -0.0002777777778,
+                   'CRPIX1': 1, 'CRVAL1': 9.423508457380343,
+                   'CTYPE2': 'DEC--TAN', 'CUNIT2': 'deg', 'CDELT2': 0.0002777777778,
+                   'CRPIX2': 1, 'CRVAL2': -33.71313112382379})
+        arr = np.arange(40000).reshape(200, 200)
+        ndd = NDData(arr, wcs=wcs)
+        imviz_helper.load_data(ndd)
+        imviz_helper.link_data(align_by='wcs')
+
+        subset_plugin = imviz_helper.plugins['Subset Tools']
+        export_plugin = imviz_helper.plugins['Export']._obj
+
+        subset_plugin.import_region(CircularROI(xc=40, yc=50, radius=10))
+        export_plugin.subset.selected = 'Subset 1'
+        export_plugin.subset_format.selected = 'stcs'
+        export_plugin.filename_value = 'Subset_1.stcs'
+        export_plugin.export()
+        with open('Subset_1.stcs') as f:
+            contents = f.read()
+        assert contents.startswith('CIRCLE')
+        assert '9.157221' in contents
+        assert '-33.435070' in contents
+        assert '0.055554' in contents
+
+        subset_plugin.import_region(EllipticalROI(xc=45, yc=55, radius_x=8, radius_y=4, theta=0.2))
+        export_plugin.subset.selected = 'Subset 2'
+        export_plugin.subset_format.selected = 'stcs'
+        export_plugin.filename_value = 'Subset_2.stcs'
+        export_plugin.export(overwrite=True)
+        with open('Subset_2.stcs') as f:
+            contents = f.read()
+        assert contents.startswith('ELLIPSE')
+        assert '9.124032' in contents
+        assert '-33.40721' in contents
+        assert '0.088886' in contents
+        assert '0.044443' in contents
+        assert '11.625269' in contents
```

Comment:
Let's we call `contents = f.read().split()`, and let's use the asserts to ensure that the each assert applies to the correct index in the `contents`, since order matters.